mixed_pond_experimental/energy_flux.py

```diff
@@ -68,20 +70,25 @@ def read_dataline(day_argue):
 def calculate_phi_sn(day_argue, hour_period):
     daily_data = read_dataline(day_argue)
     wind_speed = daily_data['wind_speed_2m']
-    #solar_daily?
-    
     
     lambda_value = lambda_s[hour_period] #remember hour period integer corresponds to lamba value
-    R_s = 2.2*((180*lambda_value)/pi)**(-0.97)
     
+
+    if lambda_value == 0:
+            R_s = 0
+    else:
+            R_s = 2.2*((180*lambda_value)/pi)**(-0.97)
+        
+
     W_z = wind_speed * 3.6
```

Comment:
Where does th 3.6 value come from? The equation in Culberson & Piedrahita it does not have this so I may be missing something?

Reply:
Oh it's to convert the wind speed from m/s to km/hr i think (per previous code) But there is another place where we need the units to be m/hr and another where it's km/hr..could you please check and let me know i may have messed up that conversion

Reply:
in C&P it shows the equation the have wind speed = wind velocity at height z meters above pond surface (m/s). (on page 254 I see this). Where did we find that it should be in km/hr?

I realize I did not look into this carefully enough the first go around sorry about that.